Makefile

```diff
@@ -139,6 +139,22 @@ clean:
 	@rm -rf test/e2e/log
 	@rm -rf e2e.namespace
 
+V1ALPHA1_GEN_PATHS := ./pkg/api/apis/operators/v1alpha1/...
+V1_GEN_PATHS := ./pkg/api/apis/operators/v1/...
+CRD_TEMPLATES :=./deploy/chart/templates
+# Generate manifests for CRDs
+manifests: ensure-controller-gen
```

Comment:
I think this should either be called by the `make codegen` command or just be included directly